tests/test_fits.py

```diff
@@ -79,5 +80,31 @@ def test_fits_reading():
     im = imageio.imread(compressed)
     assert im.shape == (2042, 3054)
 
+@pytest.mark.skipif("astropy is None")
+def test_fits_get_reader():
+    """Test reading fits with get_reader method"""
```

Comment:
Could you mention the GitHub issue (#636 ) that resulted in this regression test in the above comment? This way, it will be clear that it is a regression test, and we also have a reference to its origin,

Reply:
Just made this change in the most recent commit.